packages/tabs/addon/components/nucleus-tabs/tab-list-item.js

```diff
@@ -0,0 +1,320 @@
+import Component from '@ember/component';
+import { classNames, attributeBindings, classNameBindings, tagName, layout as templateLayout } from '@ember-decorators/component';
+import layout from '../../templates/components/nucleus-tabs/tab-list-item';
+import { get, set, computed } from '@ember/object';
+import defaultProp from '@freshworks/core/utils/default-decorator';
+import { TABS_KEY_CODE } from '../../constants/nucleus-tabs'
+
+/**
+  __Usage:__
+
+  [Refer component page](/docs/components/nucleus-tabs)
+
+  @class Nucleus Tab List Item
+  @namespace Components
+  @extends Ember.Component
+  @private
+*/
+@tagName('div')
+@templateLayout(layout)
+@classNames('nucleus-tabs__list__item')
+@classNameBindings('isActive:is-active')
+@classNameBindings('isPressed:is-pressed')
+@classNameBindings('isDisabled:is-disabled')
+@attributeBindings('tabindex')
+@attributeBindings('role')
+@attributeBindings('aria-controls')
+@attributeBindings('aria-selected')
+@attributeBindings('isDisabled:disabled')
+@attributeBindings('title')
```

Comment:
Please refactor to single binding calls.

Reply:
Done.